docs/.vuepress/config.js

```diff
@@ -15,6 +15,12 @@ const youtubeEmbed = (id, path) => `
 
 const themeColor = "#211b24"
 
+module.exports = {
+  themeConfig: {
+    searchPlaceholder: 'Search or jump to...'
```

Comment:
Concept ACK, but I don't really like `or jump to...`

maybe this?

```suggestion
    searchPlaceholder: 'Search the docs...'
```

Reply:
The fact is that you don't really search but you literally "jump" to pages which titles (h2,h3) corresponds to the keyword. 

https://github.com/zkSNACKs/WasabiDoc/issues/38

Reply:
yes, I understand the rational, but I just don't like how it sounds...